### ISSUE TITLE
Transient Revert Commands Not Showing Up

### ISSUE THREAD

**Issue by aschaap** [magit/magit#4085]
I'm sorry if I'm missing something obvious - I enjoy using Magit but have much to learn.

There was an unstashed change in my file. I tried reverting a past commit via the commit log view and _V V_, as per https://magit.vc/manual/magit/Reverting.html. The revert failed, and `magit-process` (_$_) reported local changes that would be overwritten by the merge. After removing the local changes, I tried to revert once more. This time, `magit-process` reported a revert was already in progress. However, trying _V A_ (as seen in the documentation) was not a valid option. I tried in `magit-status`, `magit-log`, and `magit-process`, and each time the transient commands available were _V V_ and _V v_. Running `git revert --continue` as suggested by the hint in the error appears to have added the conflicts into my file.

According to M-x `list-packages`, I'm using Magit 20191123.1420 within Spacemacs 0.300.0 on Emacs 26.3 (Gentoo Linux).

Let me know if there is any other information I can provide.

TL:DR; had an unstashed change, tried to revert and fail, and had no option to resume the revert.

**Comment by kyleam** [comment]
The `magit-revert` transient changes depending on whether a revert is in conflict.  When it is, you should see keys for "continue", "skip", and "abort".

Without a complete example and steps to tirgger the issue, it's going to be difficult for anyone to figure out what could be going wrong on your end.  To debug this yourself, the first place to look is probably `magit-revert-in-progress-p`, which simply checks whether `$GIT_DIR/REVERT_HEAD` exists.

**Comment by kyleam** [comment]
Closing because, for the reasons I mentioned above, I don't consider this actionable at the moment.  If you can provide more information, please reopen.
